ses

```diff
@@ -288,6 +288,16 @@ if [ -e /var/log/ganesha/ganesha.log ]; then
     plugin_message "NFS-Ganesha logs copied to ceph/log/ganesha/ subdirectory"
 fi
 
+#############################################################
+section_header "iSCSI Gateway config"
+
+if [ -x /usr/bin/gwcli ]; then
+    mkdir -p $LOG/ceph/conf/iscsi
+    plugin_command "gwcli export" |
```

Comment:
@tserong Will this command be executed by the `root` user?

Just want to make sure we will not hit this - https://github.com/ceph/ceph-iscsi/blob/master/gwcli.py#L93

Reply:
Yep (supportconfig itself will bail out if not run as root)

Reply:
Thanks